### PR TITLE
soil profile (Aida's work) and evaluation metrics

### PR DESCRIPTION
**Pull request by Bijan55699**
Addition of two new soil profiles (Aida's work) and activation of evaluation period command.

### CODE REVIEW

**Comment by Bijan55699** [comment]
soil profile (Aida's work) and evaluation metrics

File: ravenpy/config/rvs.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ class RVI(RV):
     # come after `:SoilModel`
     _post_tmpl = """
     :EvaluationMetrics     {evaluation_metrics}
-    {evaluation_periods}
+    :EvaluationPeriod      {evaluation_periods}
```
Comment (huard) [inline ravenpy/config/rvs.py:253]: Unless there is a problem with the original formulation, this should not be changed. 

This is the definition of the EvaluationPeriod command:
 ```
@dataclass
class EvaluationPeriod(RavenCommand):
    """:EvaluationPeriod [period_name] [start yyyy-mm-dd] [end yyyy-mm-dd]"""

    name: str
    start: dt.date
    end: dt.date

    template = ":EvaluationPeriod {name} {start} {end}"

    def to_rv(self):
        return self.template.format(**asdict(self))
```
In practice, use something like (untested):
```
model.config.rvt.evaluation_periods = [EvaluationPeriod('wet', dt,date(1990, 1, 1), dt.date(2000, 1, 1))]
```

**Review by huard — CHANGES_REQUESTED** [review]

**Comment by huard** [comment]
@Zeitsperre Seems to be a problem with the netcdf build. No objection pulling 3.7 from the suite if that simplifies things.

**Comment by Zeitsperre** [comment]
I believe it's this issue here: https://github.com/Unidata/netcdf4-python/issues/1179. The wheel is bad on PyPI. I can add the fix for it in here at some point.

**Comment by Zeitsperre** [comment]
@huard Python3.7 is rapidly approaching [end of life](https://peps.python.org/pep-0537/), so let's do that and see if the issue stems from there. I'm going to drop it in another PR.

File: ravenpy/config/rvs.py (hunk 1 above)
Comment (huard) [reply ravenpy/config/rvs.py:253]: What do you mean by "original formulation" ? 
https://github.com/CSHS-CWRA/RavenPy/blob/427f7caf0562928db1951e42b1217727bf925c62/ravenpy/config/commands.py#L80

If there's a but with the EvaluationPeriod configuration, let's fix it in another PR.

**Pull request closed without merging**